xen/drivers/passthrough/arm/scmi_dt_maker.c

```diff
@@ -311,3 +311,16 @@ int __init scmi_dt_scan_node(struct kernel_info *kinfo, void *pfdt,
     clean_handles();
     return rc;
 }
+
+int __init scmi_dt_set_phandle(struct kernel_info *kinfo,
```

Comment:
It is strange to add a function, but not to call it. Could this commit be squashed into one that really uses this function?